packages/thumbnail-grid-block/src/types.ts

```diff
@@ -0,0 +1,77 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Color } from '@frontify/guideline-blocks-settings';
+import { BorderStyle, DesignTokenName, Radius } from '@frontify/guideline-blocks-shared';
+import { CSSProperties } from 'react';
+
+export type Thumbnail = {
+    id: string;
+    title?: string;
+    description?: string;
+    image?: string;
+    altText?: string;
+};
+
+export enum HorizontalAlignment {
+    Left = 'left',
+    Center = 'center',
+    Right = 'right',
+}
+
+export enum VerticalAlignment {
+    Start = 'start',
+    Center = 'center',
+    End = 'end',
+}
+
+export type DesignTokens = Partial<Record<DesignTokenName, CSSProperties>>;
```

Comment:
Why not just using the DesignTokens type from fondue?

Reply:
true, thanks! changed.